example/flaskr/auth.py

```diff
@@ -80,6 +80,17 @@ def register():
                 "INSERT INTO user (username, password) VALUES (?, ?)",
                 (username, generate_password_hash(password)),
             )
+            # get the generated user id
+            userId = db.execute(
+                "SELECT id FROM user WHERE username = ?", 
+                (username,),
+            ).fetchone()['id']
+            # create a default recipeBook for the new user
+            db.execute(
+                "INSERT INTO recipeBook (author_id, title, body)  \
+                VALUES (?, ?, ?)", 
+                (userId, f"{username}'s Recipe Book", "My favorite ethical recipes."),
+            )
```

Comment:
This block creates a cookbook for any new user account. So each user has at least one cookbook. I considered moving this logic to `cookbook/save_recipe`, where we could make a cookbook for the user if none exists.